R/calculate_company_alignment_metric.R

```diff
@@ -241,36 +241,13 @@ calculate_company_aggregate_alignment_tms <- function(data,
     add_technology_share_by_direction(level = level)
 
   data <- data %>%
-    dplyr::mutate(
-      net_absolute_scenario_value = sum(!!rlang::sym(target_scenario), na.rm = TRUE),
-      .by = c(
-        "group_id", "name_abcd", "scenario_source", "region", "sector",
-        "activity_unit", "year"
-      )
-    ) %>%
-    dplyr::summarise(
-      total_deviation = sum(.data$total_tech_deviation, na.rm = TRUE),
-      .by = c(
-        "group_id", "name_abcd", "scenario_source", "region", "sector",
-        "activity_unit", "year", "net_absolute_scenario_value", "direction",
-        "technology_share_by_direction"
-      )
-    ) %>%
-    dplyr::mutate(
-      alignment_metric = .data$total_deviation / .data$net_absolute_scenario_value,
-      scenario = .env$scenario
-    ) %>%
-    dplyr::select(
-      c(
-        "group_id", "name_abcd", "sector", "activity_unit", "region",
-        "scenario_source", "scenario", "year", "direction", "total_deviation",
-        "technology_share_by_direction", "alignment_metric"
-      )
+    add_net_absolute_scenario_value(target_scenario = target_scenario) %>%
+    add_total_deviation() %>%
+    calculate_company_alignment_metric(scenario = scenario)  %>%
+    dplyr::arrange(
+      .data$group_id, .data$sector, .data$name_abcd, .data$region, .data$year
```

Comment:
I know @AlexAxthelm strongly prefers lists of arguments like this to be one per line

Reply:
do you mean this?
```
dplyr::arrange(
   .data$group_id,
   .data$sector,
   .data$name_abcd,
   .data$region,
   .data$year
)
```

Reply:
yup

Reply:
changed